recipes/scikit-odes/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "scikits-odes" %}
+{% set pypiname = "scikits.odes" %}
```

Comment:
I think we may have to use the `-` in conda-forge, but I'm not sure. Seems like other scikits use `-`. Also this is scikits not scikit which seems more prevalent.

Reply:
The use of `.` vs `-` depends on whether `.` is normalised to `-` or `_` as part of packaging.

Reply:
`name` in this case is what you type in this command `conda install scikits-odes`. I don't see `.` in other feedstock names, not sure it is a valid package name for conda.

Reply:
Actually it does look like you can have a period in the name. So maybe I should change to that.